test/karma.ci.conf.js

```diff
@@ -17,8 +17,8 @@ module.exports = function(config) {
         browsers: ['Chrome_travis_ci', 'Firefox', 'PhantomJS'],
         logLevel: config.LOG_INFO,
         preprocessors: {
-            'components/**/*.js': ['coverage']
+            'src/js/**/!(*spec|*helpers).js': ['coverage']
```

Comment:
@andy-armstrong there were no issues with running them I just thought it was best practice to exclude test files from code coverage reports. If the helpers are part of the public API and used for other (non-test) things I'm fine with including them in the coverage report, but I'm still not convinced that Jasmine tests should be included.